techMail.js

```diff
@@ -34,11 +34,11 @@ module.exports = function(configuration, templatesDir) {
 
             emailTemplates(templatesDir, function(err, template) {
                 if (err) {
-                    logger.error(err);
+                    logger.error("Unable to configure the template engine... - Details:", err && err.stack || err);
                 } else {
                     template(options.template, locals, function(err, html, text) {
                         if (err) {
-                            logger.error(err);
+                            logger.error("Unable to render the template '" + options.template + "' - Details:", err && err.stack || err);
```

Comment:
should we check options.template is defined here (before doing a "+") ? Or calling logger.error with multiple arguments (I think it should work...)